### ISSUE TITLE
Time to remove the old circular kernel indicator in the notebook toolbar?

### ISSUE THREAD

**Issue by ellisonbg** [jupyterlab/jupyterlab#10656]
### Problem

The classic Jupyter notebook has always had a circular icon in the toolbar that fills when the kernel is working. While this has been there from the beginning (I am responsible for adding it), it is not obvious to users what this is and how it works. In JupyterLab, we now have a status bar at the bottom of the page that indicate the kernel status in a more verbose, textual manner. Based on this, I am wondering if it makes sense to remove the old, curcular kernel status indicator completely.

### Proposed Solution

Remove the existing circular kernel icon in the notebook toolbar.

**Comment by telamonian** [comment]
Currently we show the "kernel is busy" status in places accross the UI:

- In plain english in the status part
- an asterix to the left of the running cell
- an hourglass favicon in the browser tab (though that assumes tabs are visible)
- the empty/filled circle indicator in question

I agree with @ellisonbg that the filled circle is somewhat cryptic and may now be redundant; my question then though is do we still need three other busy status indicators?

**Comment by krassowski** [comment]
A single user perspective:

> - In plain english in the status part

This is not a replacement for icon on the notebook toolbar:
- it only shows the status of one (focused) notebook; I often have long running computations in another notebook open next to it and want to see whether it finished without having to switch to it,
- statusbar can be hidden,
- while I personally don't hide it, statusbar is the area of the screen that is inconvenient for me to direct my vision at on a large monitor (so I don't use it unless I need to change or debug something).

> - an asterix to the left of the running cell

- asterisks are only visible if your notebook is scrolled to show the running cell. I often have "long" running computations while working a few cells down below drafting code for the next analysis step; I want to know when **all** cells below finished and the asterisks are of no use to me in this situation; however
- asterisks are extremely useful to determine status of specific cells which is as often needed as the overall status.

> - an hourglass favicon in the browser tab (though that assumes tabs are visible)

I never see it. I don't use JupyterLab in the browser (well _technically_ I do... but it is more like an Electron app for me), but as an application and the favicon changes neither propagate to the application icon in my windowing system, nor would those be visible if they did. I understand that it is nice having for users who do use it in browsers/single document mode/retrolab or whatever else.

> - the empty/filled circle indicator in question

I developed a visceral instinct to look into this area of the notebook to see the status; the change between full circle and an empty circle and vice versa is distinct enough to be seen and not too distracting either. It would be very painful for me to unlearn this and find a substitute replacement (which would be basically me writing another extension which I am very much trying to stop doing). I am happy for changing this icon to something else (assuming the new pair of icons make a good balance between being noticeable when they switch with peripheral vision and simultaneously not too flashy/distracting), but removing this indicator would be a pain for me (and I think there might be many other users like me).

**Comment by vidartf** [comment]
I'm -1 mostly for the same reasons that @krassowski added.

**Comment by aiqc** [comment]
I rely mostly on the browser tab favicon, but still find myself falling back to the blinky circle; especially when my kernel is behaving unexpectedly.

In the roadmap, we have "Progress bar or other indicator for long running notebook"
https://github.com/jupyterlab/jupyterlab/issues/8439

**Comment by jtpio** [comment]
Closing as fixed by https://github.com/jupyterlab/jupyterlab/pull/10730.
